diff --git a/dist/exports/AppRegistry/renderApplication.js b/dist/exports/AppRegistry/renderApplication.js
index b53dff630be6cba8b878aa19122931f141f8ba39..136577d2939e35e74cea1587e2bc1299fd099b44 100644
--- a/dist/exports/AppRegistry/renderApplication.js
+++ b/dist/exports/AppRegistry/renderApplication.js
@@ -11,15 +11,14 @@ import _extends from "@babel/runtime/helpers/extends";
 
 import AppContainer from './AppContainer';
 import invariant from 'fbjs/lib/invariant';
-import renderLegacy, { hydrateLegacy, render, hydrate } from '../render';
+import render, { hydrate } from '../render';
 import StyleSheet from '../StyleSheet';
 import React from 'react';
 export default function renderApplication(RootComponent, WrapperComponent, callback, options) {
   var shouldHydrate = options.hydrate,
     initialProps = options.initialProps,
-    mode = options.mode,
     rootTag = options.rootTag;
-  var renderFn = shouldHydrate ? mode === 'concurrent' ? hydrate : hydrateLegacy : mode === 'concurrent' ? render : renderLegacy;
+  var renderFn = shouldHydrate ? hydrate : render;
   invariant(rootTag, 'Expect to have a valid rootTag, instead got ', rootTag);
   return renderFn(/*#__PURE__*/React.createElement(AppContainer, {
     WrapperComponent: WrapperComponent,
diff --git a/dist/exports/findNodeHandle/index.js b/dist/exports/findNodeHandle/index.js
deleted file mode 100644
index e997fcfc68e3ddb5dbed8980ad6461c38c6f76c8..0000000000000000000000000000000000000000
diff --git a/dist/exports/findNodeHandle/index.js.flow b/dist/exports/findNodeHandle/index.js.flow
deleted file mode 100644
index 412af49c806178cd8ae81268c02b8705f268ff41..0000000000000000000000000000000000000000
diff --git a/dist/exports/render/index.js b/dist/exports/render/index.js
index aa91a2aad4d0b014735bb0efe600536715fdf6e2..c9931ef2d95501907e066f48a5c548b5253b8ee6 100644
--- a/dist/exports/render/index.js
+++ b/dist/exports/render/index.js
@@ -9,35 +9,15 @@
 
 'use client';
 
-import { hydrate as domLegacyHydrate, render as domLegacyRender } from 'react-dom';
 import { createRoot as domCreateRoot, hydrateRoot as domHydrateRoot } from 'react-dom/client';
-import unmountComponentAtNode from '../unmountComponentAtNode';
 import { createSheet } from '../StyleSheet/dom';
 export function hydrate(element, root) {
   createSheet(root);
   return domHydrateRoot(root, element);
 }
-export function render(element, root) {
+export default function render(element, root) {
   createSheet(root);
   var reactRoot = domCreateRoot(root);
   reactRoot.render(element);
   return reactRoot;
 }
-export function hydrateLegacy(element, root, callback) {
-  createSheet(root);
-  domLegacyHydrate(element, root, callback);
-  return {
-    unmount: function unmount() {
-      return unmountComponentAtNode(root);
-    }
-  };
-}
-export default function renderLegacy(element, root, callback) {
-  createSheet(root);
-  domLegacyRender(element, root, callback);
-  return {
-    unmount: function unmount() {
-      return unmountComponentAtNode(root);
-    }
-  };
-}
\ No newline at end of file
diff --git a/dist/exports/unmountComponentAtNode/index.js b/dist/exports/unmountComponentAtNode/index.js
index 925051c1c2228fd5698e14f0f3b0aef09799b102..cd92f0d7764dacea2b4b7aa14c3307fd6c0a1401 100644
--- a/dist/exports/unmountComponentAtNode/index.js
+++ b/dist/exports/unmountComponentAtNode/index.js
@@ -7,5 +7,7 @@
  * 
  */
 
-import { unmountComponentAtNode } from 'react-dom';
-export default unmountComponentAtNode;
\ No newline at end of file
+export default function unmountComponentAtNode(rootTag) {
+  rootTag.unmount();
+  return true;
+}
diff --git a/dist/index.js b/dist/index.js
index 4f699876d1902f758c3af4df5438e8ecc78bd6b4..8618c9a36421f1a57dec20eefd7f2074e325d8a6 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,5 +1,4 @@
 export { default as unstable_createElement } from './exports/createElement';
-export { default as findNodeHandle } from './exports/findNodeHandle';
 export { default as processColor } from './exports/processColor';
 export { default as render } from './exports/render';
 export { default as unmountComponentAtNode } from './exports/unmountComponentAtNode';
diff --git a/dist/vendor/react-native/VirtualizedList/index.js b/dist/vendor/react-native/VirtualizedList/index.js
index 1f52b73724ad93642909dbd5af6957a2f60ad90e..e5618992adb5890fd11fc2670e0c1e4488c63804 100644
--- a/dist/vendor/react-native/VirtualizedList/index.js
+++ b/dist/vendor/react-native/VirtualizedList/index.js
@@ -15,7 +15,6 @@ import RefreshControl from '../../../exports/RefreshControl';
 import ScrollView from '../../../exports/ScrollView';
 import View from '../../../exports/View';
 import StyleSheet from '../../../exports/StyleSheet';
-import findNodeHandle from '../../../exports/findNodeHandle';
 import Batchinator from '../Batchinator';
 import clamp from '../Utilities/clamp';
 import infoLog from '../infoLog';
